src/python/pants/engine/target.py

```diff
@@ -2466,10 +2473,35 @@ def unevaluated_transitive_excludes(self) -> UnparsedAddressInputs:
         )
 
 
+# If this predicate returns false, then the target's field's deps will be ignored.
+ShouldTraverseDepsPredicate = Callable[[Target, Field], bool]
+
+
+# noinspection PyUnusedLocal
```

Comment:
I don't see this used anywhere else in the codebase. Can you switch to one of the standard pragmas?